src/utils/commitFile.ts

```diff
@@ -30,12 +25,14 @@ if (platform !== "win32" && platform !== "darwin" && platform !== "linux") {
 export const commitFileMac = async (
   owner: string,
   repo: string,
-  refs: string
+  refs: string,
+  authToken: string
 ): Promise<response> => {
   let gitCommands: commands;
   let index: number;
   let isCodespace = false as boolean;
 
+  const authBaseURL = baseURL!.replace("https://", `https://x-access-token:${authToken}@`) as string;
```

Comment:
👋 what is the justification behind this? 🤔 Shouldn't this only be needed for unattended workflows (as the user isn't logged into the machine). Are you doing this to keep it consistent across the different environments?

Reply:
👍 I chose to put it here to easily be consistent over environments indeed. 

I thought that even if you run it manually/attended, it is probably fine if the push is done with whatever auth mechanism you configured instead of your personal SSH key or password (personally that is how I expected it would behave). 

I believe the app will not run without either a PAT or GitHub app credentials set (as remote branch and pr are created through the api), so I don't think that ruling out the use of ssh-key/password for this step would limit functionality. 

I am not sure how to safely check that we are running unattended, so if it this needs to be optional, my first thought would be to add an extra environment parameter to set unattended execution.
(I think I misinterpreted your answer to my original question about this 😅)

Please advise :D

Reply:
This is fine 👍 I just wanted to clarify 👍 I have no problems with the way you have done it 👍 Just wanted to understand for my own knowledge 🙇